scripts/run-framework/run-framework.py

```diff
@@ -133,6 +133,9 @@ def setup_arg_parsing():
                         help='Logging level (default=%(default)s). '
                              'Options: debug, info, warning, error, critical')
 
+    parser.add_argument('--no_compress', dest='no_compress', action='store_true',
```

Comment:
With reference to my earlier comment about variable name, I think if it is called no_compress, it implies that the default is to compress. How do you think about it?